src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/troubleshooting/collector-issues.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Troubleshoot OpenTelemetry: Collector issues"
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+  - Troubleshooting
+metaDescription: Troubleshoot tips for when you use OpenTelemetry with New Relic
+freshnessValidatedDate: 2024-05-01
+---
+
+## Problem
+
+You’re using an OpenTelemetry collector to send data to New Relic, but you’re experiencing issues like:
+* The collector exits or restarts
+* The collector is not receiving data
+* New Relic is not receiving data
+* Processing/exporting data is not working
+
+
+## Solution
+
+For solutions to common collector problems, see the [OpenTelemetry troubleshooting documentation](https://opentelemetry.io/docs/collector/troubleshooting/).
+
+Within New Relic, we recommend using this NRQL query to show all collector metrics sent to New Relic:
```

Comment:
What are some examples of things they can find/troubleshoot with this data?

Reply:
Some of the metrics the query below pertains to are documented here https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/monitoring.md.